core/modules/xrdsvc/ChannelStream.cc

```diff
@@ -39,16 +39,19 @@ namespace {
 LOG_LOGGER _log = LOG_GET("lsst.qserv.xrdsvc.ChannelStream");
 }
 
+using namespace std;
 
 namespace lsst {
 namespace qserv {
 namespace xrdsvc {
 
+/// Provide each Channel stream with a unique identifier.
+static atomic<uint32_t> localSeq(0);
```

Comment:
Why this is here, rather than being declared as a static member of class `ChannelStream` where it belongs to?